src/Illuminate/Database/Console/Seeds/SeedCommand.php

```diff
@@ -88,6 +94,29 @@ protected function getDatabase()
         return $database ?: $this->laravel['config']['database.default'];
     }
 
+    /**
+     * Truncate all tables in database.
+     *
+     * @return void
+     */
+    protected function truncateTables()
+    {
```

Comment:
please add a return type to the method 
```PHP
 protected function truncateTables() : void
```

Reply:
@TBlindaruk I'm pretty sure these return types are not enforced anywhere within the framework, regardless of 
 5.6 requiring PHP 7.1+

Reply:
Yes, laravel doesn't use scalar types or return types.